### PR TITLE
FIX: news page error while scolling, header view for admin

### PR DESCRIPTION
**Pull request by ArtoNazaryan**
### What are you trying to accomplish?
fixed news page scroll error bug, header notifications and buttons (exept logout) are hidden for admin now.

### CODE REVIEW

File: src/components/Header/Header.tsx [hunk 1]
```diff
@@ -28,10 +28,10 @@ export const Header = () => {
     },
     { name: t(Translation.NAVBAR_PROFILE), href: '/profile/settings', current: false },
   ]
-
   const isAuthenticated = useAppSelector(usersSelector.isAuthenticated)
   const isLogOutLoading = useAppSelector(usersSelector.isLogOutLoading)
-
+  const { role } = useAppSelector(usersSelector.user)
+  const checkIfUserAdmin: boolean = role !== Roles.Admin && role !== Roles.NOROLE
```
Comment (ArthurO2002) [inline src/components/Header/Header.tsx:34]:
```suggestion
  const isAdmin: boolean = role !== Roles.Admin && role !== Roles.NOROLE
```

File: src/components/Header/User.tsx [hunk 2]
```diff
@@ -22,6 +23,8 @@ export const User = ({ handleLogOut }: IHeaderUserProps) => {
     { href: '/profile/settings', label: t(Translation.NAVBAR_USER_MENU_EDIT_PROFILE) },
   ]
 
+  const checkIfUserAdmin: boolean = role !== Roles.Admin && role !== Roles.NOROLE
```
Comment (Antonyan7) [inline src/components/Header/User.tsx:26]: isAdmin?

**Review by Antonyan7 — CHANGES_REQUESTED** [review]

File: src/components/Header/User.tsx (hunk 2 above)
Comment (ArthurO2002) [inline src/components/Header/User.tsx:26]:
```suggestion
  const isAdmin: boolean = role !== Roles.Admin && role !== Roles.NOROLE
```

File: src/components/Header/User.tsx [hunk 3]
```diff
@@ -51,21 +54,23 @@ export const User = ({ handleLogOut }: IHeaderUserProps) => {
...
-                            variant="outlined"
-                            onClick={() => close()}
-                          >
-                            {link.label}
-                          </Button>
-                        </Link>
-                      </li>
-                    </Menu.Item>
-                  ))}
+                  {checkIfUserAdmin
```
Comment (ArthurO2002) [inline src/components/Header/User.tsx:57]:
```suggestion
                  {isAdmin
```

**Review by Antonyan7 — COMMENTED** [review]
LGTM except one small thing

**Review by ArthurO2002 — APPROVED** [review]

**Pull request merged**